common/test/db_cache/bok_choy_schema.sql

```diff
@@ -20,8 +20,8 @@ CREATE TABLE `assessment_aiclassifier` (
   PRIMARY KEY (`id`),
   KEY `assessment_aiclassifier_962f069f` (`classifier_set_id`),
   KEY `assessment_aiclassifier_385b00a3` (`criterion_id`),
-  CONSTRAINT `D3bd45d5e3c9cfdc4f3b442119adebe8` FOREIGN KEY (`classifier_set_id`) REFERENCES `assessment_aiclassifierset` (`id`),
-  CONSTRAINT `assessm_criterion_id_275db29f2a0e1711_fk_assessment_criterion_id` FOREIGN KEY (`criterion_id`) REFERENCES `assessment_criterion` (`id`)
+  CONSTRAINT `assessm_criterion_id_275db29f2a0e1711_fk_assessment_criterion_id` FOREIGN KEY (`criterion_id`) REFERENCES `assessment_criterion` (`id`),
```

Comment:
Yes.  They help speed up bok choy tests, but need to be updated  due to the migration funkiness in this PR.